code/__HELPERS/honkerblast.dm

```diff
@@ -0,0 +1,47 @@
+///Unleashes a honkerblast similar to the honkmech weapon, but with more granular control.
```

Comment:
why not make the honkmech weapon use this proc, then?

Reply:
Well i didn't want to get into the honk mech balance discussion on this PR.